src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Mortsel.cs

```diff
@@ -23,6 +23,14 @@ public override IList<HouseNumberWithBoxNumber> GetValues()
                     new HouseNumberWithBoxNumber(
                         HouseNumberSourceValue,
                         "glv"
+                    ),
+                    new HouseNumberWithBoxNumber(
+                        HouseNumberSourceValue,
+                        "glvl"
+                    ),
+                    new HouseNumberWithBoxNumber(
+                        HouseNumberSourceValue,
+                        "GLVL"
```

Comment:
When matching addresses, we ignore cases, see https://github.com/Informatievlaanderen/basisregisters-integration-db/blob/main/src/Basisregisters.IntegrationDb.NationalRegistry/AddressMatching/AddressMatchRunner.cs#L66

Reply:
Fixed